katas/content/superdense_coding/alice_sends_message/Placeholder.qs

```diff
@@ -0,0 +1,12 @@
+namespace Kata {
+    newtype ProtocolMessage = (Bit1 : Bool, Bit2 : Bool);
+
+    operation EncodeMessageInQubit(qAlice : Qubit, message : ProtocolMessage) : Unit {
```

Comment:
Looking at this kata in the modern system, I think we should drop ProtocolMessage type altogether, and rewrite it as just a tuple of two bits, like this:
```suggestion
    operation EncodeMessageInQubit(qAlice : Qubit, (bit1, bit2) : (Int, Int)) : Unit {
```

It will make the code much shorter and more straightforward, for example, we won't have to teach bit access in the kata. And in this task it makes things confusing, actually, since we're asking them to implement solution within an if that is supposed to teach bit access, but the solution also needs to include another, independent if outside of this one... Plus then we can drop protocol notes in task description

Reply:
Reverted to tuple of Booleans by checking out to previous commit.